Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -529,7 +529,7 @@ public void Actions_CanTargetSameControlWithMultipleActions()
     {
 #if UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
         // Exclude project-wide actions from this test
-        InputSystem.actions?.Disable();
+        InputSystem.actions?.Disable(); // Prevent these actions appearing in the `InputActionTrace`
```

Comment:
Great to see the extra comments explaining why disabled.
I did wonder if we need some re-enabling?

Reply:
All tests are enabled in this PR.
All tests have project-wide actions enabled where appropriate.
Everywhere project-wide actions is disabled has been reviewed and a note added for the rationale.